src/lib.rs

```diff
@@ -83,9 +83,9 @@ for entry in WalkDir::new("foo").follow_links(true) {
 
 [`follow_links`]: struct.Walkdir.html#method.follow_links
 
-# Example: skip hidden files and directories efficiently on unix
+# Example: skip hidden files and directories on unix
```

Comment:
I used the word "efficiently" here to imply something, namely, then it will avoid recursing into hidden directories. Maybe that should be more explicitly stated.